tools/msstatstmt/msstatstmt.xml

```diff
@@ -0,0 +1,379 @@
+                <param name="oms_input" type="data" format="tabular" label="OpenMS input"/>
+                <section name="input_options" title="OpenMStoMSstatsTMTFormat Options" expanded="false">
+                    <expand macro="input_options_shared"/>
+                    <param name="summaryforMultiplePSMs" type="select" label="Summary for multiple PSMs" help="When there are multiple measurements for certain feature in certain run, select the feature with the largest summation or maximal value.">
+                        <option value="max">max</option>
+                        <option value="sum" selected="true">sum</option>
+                    </param>
+                </section>
+            </when>
+        </conditional>
+        <section name="proteinSummarization" title="proteinSummarization Options" expanded="false">
+                <param name="method" type="select" multiple="false" label="Select method">
+                    <option value="msstats" selected="true">msstats</option>
+                    <option value="MedianPolish">MedianPolish</option>
+                    <option value="Median">Median</option>
+                    <option value="LogSum">LogSum</option>
+                </param>
+                <param name="global_norm" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" label="Global median normalization" help="Global median normalization on peptide level data (equalizing the medians across all the channels and MS runs). Default is Yes. It will be performed before protein-level summarization."/>
+                <param name="reference_norm" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" label="Reference channel based normalization" help="Reference channel based normalization between MS runs on protein level data. Yes (default) needs at least one reference channel in each MS run, annotated by ’Norm’ in condition column. It will be performed after protein-level summarization. No will not perform this normalization step. If data only has one run, then use No"/>
+                <param name="remove_norm_channel" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" label="Remove ’Norm’ channels from protein level data."/>
+                <param name="remove_empty_channel" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" label="Remove ’Empty’ channels from protein level data"/>
+                <param name="MBimpute" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="true" label="MBimpute" help="Only for 'method = msstats'. Yes (default) imputes missing values by accelerated failure time (AFT) model. No uses minimum value to impute the missing value for each peptide precursor ion."/>
+                <conditional name="maxQuantileforCensored">
+                    <param name="select" type="select" label="Maximum quantile for deciding censored missing value">
+                        <option value="false">No</option>
```

Comment:
We assume missing values are censored. maxQuantileforCensored is Maximum quantile for deciding censored missing value, for instance, 0.999. Default is Null.
--> I assume this means more an optional float

Reply:
Thank you for pointing this out! It's much simpler now..